rollup/sync_service/bridge_client.go

```diff
@@ -36,61 +44,106 @@ func newBridgeClient(ctx context.Context, l1Client EthClient, l1ChainId uint64,
 		return nil, fmt.Errorf("unexpected chain ID, expected = %v, got = %v", l1ChainId, got)
 	}
 
-	filterer, err := NewL1MessageQueueFilterer(l1MessageQueueAddress, l1Client)
+	filtererV1, err := NewL1MessageQueueFilterer(l1MessageQueueV1Address, l1Client)
+	if err != nil {
+		return nil, fmt.Errorf("failed to initialize L1MessageQueueV1Filterer, err = %w", err)
+	}
+	filtererV2, err := NewL1MessageQueueFilterer(l1MessageQueueV2Address, l1Client)
 	if err != nil {
-		return nil, fmt.Errorf("failed to initialize L1MessageQueueFilterer, err = %w", err)
+		return nil, fmt.Errorf("failed to initialize L1MessageQueueV2Filterer, err = %w", err)
 	}
 
 	client := BridgeClient{
-		client:                l1Client,
-		confirmations:         confirmations,
-		l1MessageQueueAddress: l1MessageQueueAddress,
-		filterer:              filterer,
+		client:        l1Client,
+		confirmations: confirmations,
+
+		l1MessageQueueV1Address: l1MessageQueueV1Address,
+		filtererV1:              filtererV1,
+
+		l1MessageQueueV2Address: l1MessageQueueV2Address,
+		filtererV2:              filtererV2,
```

Comment:
I was thinking we can just use a single filter with to addresses. But this way also seems OK.